homework_2/fastq_filtrator.py

```diff
@@ -0,0 +1,69 @@
+
+def gc_content(read, threshold):
+    read = read.upper()
+    gc_number = 0
+    gc_number += read.count('G')
+    gc_number += read.count('C')
+    gc_percentage = (gc_number / len(read)) * 100
+
+    if type(threshold) is int:
+        high_border = threshold
+        return gc_percentage < high_border
+
+    if type(threshold) is tuple:
+        low_border = threshold[0]
+        high_border = threshold[1]
+        return low_border < gc_percentage < high_border
```

Comment:
Пороговые значения везде нужно было включать
```suggestion
        return low_border <= gc_percentage <= high_border
```